### ISSUE TITLE
Rotated images appear not rotated in uploading preview

### ISSUE THREAD

**Issue by Aktron** [yarl/pattypan#118]
If you are reporting a bug please:

 - Check the bug has already been reported (https://github.com/yarl/pattypan/issues).
 - Explain the current issue.
 - Describe the expected behavior.
 - Add the problematic spreadsheet and a example media file (drag & drop here).
 - Add the PattyPan log file (drag & drop here). You find a link to the PattyPan log at the application start screen.

**Comment by Abbe98** [comment]
Yeah, it behaves similar to how UploadWizard does it, but is indeed a bug.
